src/components/video-player-theme.css

```
@@ -0,0 +1,353 @@
+/**
+ * Video Player Theme (media-chrome)
+ * Scoped to media-controller. Uses project color variables.
+ */
+@reference "../styles/globals.css";
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@reference" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)